config/kuzzlerc

```diff
@@ -0,0 +1,14 @@
+{
+  "plugins": {
+    "s3": {
+      "bucketName": "your-s3-bucket",
+      "region": "eu-west-3",
+      "signedUrlTTL": "20min",
+      "redisPrefix": "s3Plugin/uploads",
+      "vault": {
+        "accessKeyIdPath": "toto.s3.accessKeyId",
```

Comment:
prefer `foo` if you need a placeholder value, it's more universal and understandable